survey_resource_booking/models/resource_booking.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2021 Tecnativa - Jairo Llopis
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, fields, models
+from odoo.tests.common import Form
+from odoo.tools.mail import email_split
+from odoo.exceptions import UserError
+
+
+class ResourceBooking(models.Model):
+    _inherit = "resource.booking"
+
+    survey_user_input_id = fields.Many2one(
+        "survey.user_input",
+        string="Survey user input",
+        track_visibility="onchange",
```

Comment:
Why not being this field `readonly`? You can always fix any mess through shell, mass editing...

Reply:
It's because, if for any reason you want to invite the requester to fill the survey again (invalid input? lost invitation? etc.), you have to empty this field before. Otherwise, you'll get an error telling you that he was already invited.

It could have been done by other means, but at least that's how it is implemented.

Reply:
That's a bit ugly and the invite method should handle it in my opinion. Isn't a code like this enough:
```python
   if self.survey_user_input_id:
       self.survey_user_input_id = False
```

Reply:
What do you think about this? At least a reference in the ROADMAP?

Reply:
It's just a different way of supporting something that is right now supported. The customer never complained about this, so I'd just leave things as they are.